packages/marketplace-ui/package.json

```diff
@@ -56,6 +59,7 @@
     "react-hook-form": "^7.25.1",
     "react-hot-toast": "^2.2.0",
     "recharts": "^2.1.9",
+    "request-promise": "^4.2.6",
```

Comment:
Looks like this package is deprecated and also the package `request` it depends on is deprecated?